src/modules/LikeButton.js

```diff
@@ -0,0 +1,26 @@
+const getLikes = async () => {
+  const uniqueID = 'koWShQRS6nOY80YNyou5';
```

Comment:
The application works very well 👌but security is also an important concept, so please make sure to not store the secret keys inside your repository. I can recommend two solutions, but feel free to use whatever works best for you.
1. You can use a `.env` file
2. You can move your secret keys to another js file and import that in here as a variable and that file must be added to the `.gitignore` file.

And at the end please make sure to add the guidelines in your Readme file, so that the user can understand how to setup this project properly.